config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -277,7 +277,7 @@ periodics:
       # kind specific args
       - --provider=skeleton
       - --deployment=kind
-      - --kind-binary-version=stable
+      - --kind-binary-version=build
```

Comment:
I forget how we implemented this, but it might not be sufficient to actually get HEAD, go modules is going to prefer tags unless you specify otherwise.

Reply:
i think HEAD used to work before we switched to 0.2.0.

Reply:
go get with modules enabled will get the latest tag by default, in any case 0.4 is about to come out.

Reply:
if i recall correctly in the jobs and deployer, we are cloning k-sigs/kind and building from that latest SHA, by calling `go build` in the kind path.